### PR TITLE
fix: deduplicate env rpc endpoint and other minor improvements

### PR DESCRIPTION
**Pull request by kilted-andres**
- Deduplicates environment constant for rpc endpoint => unifies `BLOCKCHAIN_ENDPOINT` and `POLKADOT_RPC_ENDPOINT`.
- Changes the `QUERY_SIZE` from **50** to **100** that is the maximum of the _indexer_.
- Handles case of bad cType written on chain (and on the indexer).

### CODE REVIEW

**Review by ggera — APPROVED** [review]

File: src/components/CTypeDetails/CTypeDetails.astro [hunk 1]
```diff
@@ -99,7 +99,7 @@ const { w3nOrigin, indexer } = configuration;
     <legend>Registration Block</legend>
 
     <a
-      href={`https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F${indexer.polkadotRPCEndpoint}#/explorer/query/${block}`}
+      href={`https://polkadot.js.org/apps/?rpc=${blockchainEndpoint}#/explorer/query/${block}`}
```
Comment (arty-name) [inline src/components/CTypeDetails/CTypeDetails.astro:102]: I highly recommend against building URLs this way. I was not pushing against it when you were only adding ASCII characters previously. Now when you inject special characters, you really should do either `encodeURIComponent` or build the URL using the `URL()` API.

File: src/components/CTypeDetails/__snapshots__/CTypeDetails.test.ts.snap [hunk 2]
```diff
@@ -121,7 +121,7 @@ exports[`CTypeDetails > should handle kitchen sink CType 1`] = `
   </fieldset>
   <fieldset>
     <legend>Registration Block</legend>
-    <a href="https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fplaceholder#/explorer/query/456">456</a>
+    <a href="https://polkadot.js.org/apps/?rpc=ws://localhost:32769#/explorer/query/456">456</a>
```
Comment (arty-name) [inline src/components/CTypeDetails/__snapshots__/CTypeDetails.test.ts.snap:124]: See, this is an invalid URL

File: src/utilities/configuration.ts [hunk 3]
```diff
@@ -57,11 +52,16 @@ if (!payerMnemonic) {
   throw new ConfigurationError('SECRET_PAYER_MNEMONIC is not provided');
 }
 
-const w3nOrigins: Record<string, string> = {
-  'wss://peregrine.kilt.io': 'https://test.w3n.id',
-  'wss://peregrine-stg.kilt.io/para': 'https://smoke.w3n.id',
-  'wss://kilt-rpc.dwellir.com': 'https://w3n.id',
-};
+function deductW3nOrigin(blockchainEndpoint: string) {
+  const endpoint = blockchainEndpoint.toLocaleLowerCase();
```
Comment (arty-name) [inline src/utilities/configuration.ts:56]:
```suggestion
  const endpoint = blockchainEndpoint.toLowerCase();
```
I believe locale is irrelevant here

File: src/utilities/indexer/queryCTypes.ts [hunk 4]
```diff
@@ -77,17 +77,25 @@ export async function queryCTypes() {
...
+        ...rest,
+        attestationsCreated,
+      });
+      logger.info(
+        `Added new CType to data base: ${JSON.stringify(newCType, null, 2)}`,
+      );
+    } catch (error) {
+      logger.error(
+        error,
+        `Could not add cType ${cTypeId} to database. Probably bad formatted, see it's definition: ${definition}`,
```
Comment (arty-name) [inline src/utilities/indexer/queryCTypes.ts:96]:
```suggestion
        `Could not add cType ${cTypeId} to database. Probably bad formatted, see its definition: ${definition}`,
```
Again I recommend a good spellchecker :)

**Review by arty-name — CHANGES_REQUESTED** [review]

File: src/utilities/configuration.ts (hunk 3 above)
Comment (kilted-andres) [reply src/utilities/configuration.ts:56]: Yes, it was autocomplete.

Comment (kilted-andres) [reply src/utilities/configuration.ts]: done

File: src/utilities/indexer/queryCTypes.ts (hunk 4 above)
Comment (kilted-andres) [reply src/utilities/indexer/queryCTypes.ts]: done

File: src/components/CTypeDetails/__snapshots__/CTypeDetails.test.ts.snap (hunk 2 above)
Comment (kilted-andres) [reply src/components/CTypeDetails/__snapshots__/CTypeDetails.test.ts.snap:124]: This is a valid URL. I just tried it out with a local chain.

**Review by arty-name — APPROVED** [review]

**Pull request merged**